Mathlib/Data/SetLike/Basic.lean

```diff
@@ -186,9 +186,6 @@ theorem coe_eq_coe {x y : p} : (x : B) = y ↔ x = y :=
   Subtype.ext_iff_val.symm
 #align set_like.coe_eq_coe SetLike.coe_eq_coe
 
--- porting note: this is not necessary anymore due to the way coercions work
-#noalign set_like.coe_mk
```

Comment:
I think it's good to keep the `#noalign`, until we drop all `#align` statements.

Reply:
@jcommelin Done.